expo/gbe/models/available_interest.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+
+class AvailableInterest(models.Model):
+    interest = models.CharField(
+        max_length=128,
+        unique=True)
+    visible = models.BooleanField(default=True)
+    help_text = models.TextField(max_length=500,
```

Comment:
I try to be in the habit of always limiting length.  Didn't realize textfield doesn't use it - but I see that in the DB.  will pull it.